client/src/components/Modals/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import "./style.css";
+import axios from 'axios';
+
+class MyVerticallyCenteredModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.logout = this.logout.bind(this)
+    this.state = {
+      modalShow: true
+    }
+  }
+
+  logout(event) {
+    event.preventDefault()
+    console.log('logging out')
+    axios.post('/logout').then(response => {
+      console.log(response.data)
```

Comment:
Do i need to know what the data will look like? I'm assuming the data is there and everything is working

Reply:
also missing semicolon